CMakeLists.txt

```diff
@@ -77,3 +82,37 @@ include_directories(${BTSERVICE_PROTO_INCLUDE_DIRS})
 add_subdirectory(openauto)
 add_subdirectory(autoapp)
 add_dependencies(autoapp btservice_proto)
+
+install(DIRECTORY lib DESTINATION lib COMPONENT libraries)
+install(DIRECTORY include DESTINATION include COMPONENT headers)
+install(DIRECTORY bin DESTINATION bin COMPONENT applications)
+
+SET(CPACK_GENERATOR "DEB")
+SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "OpenCarDev") #required
```

Comment:
should this (and the line below) be OpenCarDev?

i know the crankshaft openauto is aloooooot different and dont want any confusion

Reply:
Fixed the typo